pkg/cmd/cli/cmd/login/logout.go

```diff
@@ -123,25 +126,39 @@ func (o LogoutOptions) RunLogout() error {
 		return err
 	}
 
-	if err := client.OAuthAccessTokens().Delete(token); err != nil {
-		return err
+	errs := []error{}
+	serverErr := client.OAuthAccessTokens().Delete(token)
+	if serverErr != nil {
+		errs = append(errs, serverErr)
+	}
+
+	if configErr := deleteTokenFromConfig(*o.StartingKubeConfig, o.PathOptions, token); configErr != nil {
+		errs = append(errs, configErr)
+	} else if serverErr != nil {
+		// if token was not able to be deleted using api but was successfully removed
+		// from client config, print brief explanation that token was still removed locally
+		fmt.Fprintf(o.ErrOut, "%s\n\n", "An error ocurred deleting the token on the server. The token has been removed from your local configuration.")
```

Comment:
Although this also applies to cases outside of being logged in with a serviceaccount token, I am not sure how I feel about returning an error message everytime a user tries to log out when using a serviceaccount token. cc @fabianofranz